server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -63,6 +63,11 @@ public class FeatureFlags {
      */
     public static final String CONCURRENT_SEGMENT_SEARCH = "opensearch.experimental.feature.concurrent_segment_search.enabled";
 
+    /**
+     * Gates the functionality of tracing framework.
+     */
+    public static final String TRACER = "opensearch.experimental.feature.tracer.enabled";
```

Comment:
Should we specify otel here ?
```opensearch.experimental.feature.tracer.otel.enabled```
And more meaning full name this it is ```request.tracer```

Reply:
The flag is for overall tracing framework and not specific to Otel implementation. So I think it should be ok to keep it as such